ansible_rulebook/app.py

```diff
@@ -72,7 +72,7 @@ def qsize(self):
 
 
 def log_exception_without_data(exc_type, exc_value, exc_traceback):
-    logger.error(exc_type.__name__ + ": " + str(exc_value))
+    logger.error(exc_type.__name__)
```

Comment:
@kaiokmo Are we going to loose the exception details when it fails for some other issue in the code. Most of the source plugins are written by customers and if there is a bug in their code they should be able to self diagnose and fix the issues.